.github/workflows/trivy.yml

```diff
@@ -0,0 +1,29 @@
+name: dsp-appsec-trivy
+on: [pull_request]
+
+jobs:
+  appsec-trivy:
+    # Parse Dockerfile and build, scan image if a "blessed" base image is not used
+    name: DSP AppSec Trivy check
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        context:
+        - api
+        - ui
+
+    steps:
+      - uses: actions/checkout@v2
+
+      # API Dockerfile copies this, so it needs
+      # to exist for the build to succeed
+      # (even though it's not scanned)
```

Comment:
Thanks for the comment! So I guess that Java(Clojure indeed) code is covered by some sort of Source Clear and does not need to be scanned here?

Reply:
Yep, exactly! It would be helpful to move the Clojure build instruction into the Dockerfile, to make it truly reproducible and so we don't have to use this workaround :-)

Reply:
Yea, historically it's outside of Dockerfile since the build has to clone several private github repos and it's not easy to do so within a Docker. We are close to get rid of the private images and once it's done we can move into the Dockerfile! (the last step is to move the config I mentioned to AppSec to WFL's own repo)

Reply:
Sounds great!!